### PR TITLE
Please add Czech language to Language models

### PR DESCRIPTION
**Pull request by suculent**
I'm not sure, what needs to be done in pentest/migrations (if anything at all) to make this work. Thanks for guidance.

### CODE REVIEW

File: api/src/reportcreator_api/pentests/models/common.py [hunk 1]
```diff
@@ -128,6 +128,7 @@ class Language(models.TextChoices):
     ENGLISH_CA = 'en-CA', True, 'English (en-CA)'
     ENGLISH_NZ = 'en-NZ', True, 'English (en-NZ)'
     ENGLISH_ZA = 'en-ZA', True, 'English (en-ZA)'
+    CZECH = 'cs-CZ', True, 'Czech (cs-CZ)'
```
Comment (MWedl) [inline api/src/reportcreator_api/pentests/models/common.py:131]: Languagetool does not support spellchecking Czech (list of supported languages: https://dev.languagetool.org/languages). Please set the `spellcheck` argument (second item in tuple) to `False` and move the `CZECH` entry into the section below ("Languages without LanguageTool support").

**Review by MWedl — COMMENTED** [review]
Thanks for your contribution.

Migrations can be auto-generated via:
```sh
cd sysreptor/dev
docker compose run --rm api python3 manage.py makemigrations
```

**Comment by suculent** [comment]
Unfortunately this fails after start:

`
2024-10-25 10:30:49,810 [ERROR] root: 'cs-CZ' is not a valid Language
sysreptor-app           | Traceback (most recent call last):
sysreptor-app           |   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 518, in thread_handler
sysreptor-app           |     raise exc_info[1]
sysreptor-app           |   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/base.py", line 253, in _get_response_async
sysreptor-app           |     response = await wrapped_callback(
sysreptor-app           |                ^^^^^^^^^^^^^^^^^^^^^^^
sysreptor-app           |   File "/usr/local/lib/python3.12/site-packages/adrf/viewsets.py", line 120, in async_view
sysreptor-app           |     return await self.dispatch(request, *args, **kwargs)
sysreptor-app           |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sysreptor-app           |   File "/usr/local/lib/python3.12/site-packages/adrf/views.py", line 73, in async_dispatch
sysreptor-app           |     response = self.handle_exception(exc)
sysreptor-app           |                ^^^^^^^^^^^^^^^^^^^^^^^^^^
sysreptor-app           |   File "/usr/local/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
sysreptor-app           |     self.raise_uncaught_exception(exc)
sysreptor-app           |   File "/usr/local/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
sysreptor-app           |     raise exc
sysreptor-app           |   File "/usr/local/lib/python3.12/site-packages/adrf/views.py", line 70, in async_dispatch
sysreptor-app           |     response = await sync_to_async(handler)(request, *args, **kwargs)
sysreptor-app           |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sysreptor-app           |   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 468, in __call__
sysreptor-app           |     ret = await asyncio.shield(exec_coro)
sysreptor-app           |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sysreptor-app           |   File "/usr/local/lib/python3.12/site-packages/asgiref/current_thread_executor.py", line 40, in run
sysreptor-app           |     result = self.fn(*self.args, **self.kwargs)
sysreptor-app           |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sysreptor-app           |   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 522, in thread_handler
sysreptor-app           |     return func(*args, **kwargs)
sysreptor-app           |            ^^^^^^^^^^^^^^^^^^^^^
sysreptor-app           |   File "/app/api/reportcreator_api/api_utils/views.py", line 76, in settings_endpoint
sysreptor-app           |     } for l in remove_duplicates(list(map(Language, settings.PREFERRED_LANGUAGES)) + list(Language))]
sysreptor-app           |                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sysreptor-app           |   File "/usr/local/lib/python3.12/enum.py", line 757, in __call__
sysreptor-app           |     return cls.__new__(cls, value)
sysreptor-app           |            ^^^^^^^^^^^^^^^^^^^^^^^
sysreptor-app           |   File "/usr/local/lib/python3.12/enum.py", line 1171, in __new__
sysreptor-app           |     raise ve_exc
`

File: api/src/reportcreator_api/pentests/models/common.py (hunk 1 above)
Comment (suculent) [reply api/src/reportcreator_api/pentests/models/common.py]: Unfortunately the app fails with "root: 'cs-CZ' is not a valid Language" while opening the login page. I've added the language code to preferred languages:

PREFERRED_LANGUAGES="cs-CZ,de-DE,en-US,sk-SK"

**Comment by MWedl** [comment]
What SysReptor version are you using? Your changes are included in version 2024.81 that was released only some minutes ago.

**Comment by suculent** [comment]
Initially, I've rebuilt the docker image. I've pulled the repository clean now and it seems to be working. Thanks.

**Pull request merged**